com.ibm.streamsx.plumbing/com.ibm.streamsx.plumbing.synchronize/Synchronize/Synchronize.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--                                                                        -->
+<!-- Copyright (C) 2013, 2016 International Business Machines Corporation.  -->
+<!-- All Rights Reserved.                                                   -->
+<!--                                                                        -->
+<operatorModel
+  xmlns="http://www.ibm.com/xmlns/prod/streams/spl/operator" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:cmn="http://www.ibm.com/xmlns/prod/streams/spl/common"
+  xsi:schemaLocation="http://www.ibm.com/xmlns/prod/streams/spl/operator operatorModel.xsd">
+  <cppOperatorModel>
+
+    <context>
+      <description>
+
+The Synchronize operator synchronizes the flow of tuples across multiple streams according to timestamps in the tuples.  It can be configured with one or more input ports, and must have the same number of output ports.  Optionally, it can pace tuples faster or slower than they were produced, according to their timestamps.
+
+The Synchronize operator does not alter tuples that pass through it. All tuples are forwarded as received to the output port corresponding to their input port.
+
+The Synchronize operator synchronizes streams by receiving tuples in all of its input ports and forwarding them downstream in order according to their timestamps, temporarily holding tuples in faster streams until slower streams catch up.  Optionally, the operator can pace tuples by delaying them according to the differences between their timestamps, according to a time-scaling factor. 
```

Comment:
The behaviour is a little unclear:
* Does it require all input ports to receive a tuple before any tuple is forwarded to an output port (assuming monotonically increasing values)?
* 'forwarding them downstream in order according to their timestamps' - is this just in order of the timestamps, or is it paced according to the timestamp as well?

E.g. with just two input ports A and B I might get A(100) and B(1), B(2), B(3) and it seems I cannot send A(100) until I see a B(n) where N >= 100, but I might only see B(5000) which means A(100) is not sent at the correct "time" with respect to any earlier B tuples that were forwarded.